python/src/trezorlib/cli/device.py

```diff
@@ -53,12 +53,12 @@ def cli() -> None:
 
 @cli.command()
 @with_client
-def self_test(client: "TrezorClient") -> str:
-    """Perform a factory self-test.
+def prodtest_t1(client: "TrezorClient") -> str:
```

Comment:
while we're here, let's move the command to `debug` section

Reply:
https://github.com/trezor/trezor-firmware/pull/3538/commits/26c17c187235fb3d0d1c6432adf2cb7a18aaba37